rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -11,6 +11,8 @@ import ActivityList from "../activityList.vue";
 import ActivityFilter from "../activityFilter.vue";
 import OffsetPagination from "../../../../library/components/utils/OffsetPagination.vue";
 import { cloneDeep } from "lodash";
+import Modal from "uiv";
+import Btn from "uiv";
```

Comment:
Imported Modal and Btn from uiv.

Reply:
these imports are wrong, it should be:
import { Modal, Btn } from "uiv"

(when in doubts how to import a third party library, check their docs)